lib/class-wp-theme-json-gutenberg.php

```diff
@@ -217,9 +217,11 @@ class WP_Theme_JSON_Gutenberg {
 	 */
 	const PROPERTIES_METADATA = array(
 		'aspect-ratio'                      => array( 'dimensions', 'aspectRatio' ),
-		'background'                        => array( 'color', 'gradient' ),
 		'background-color'                  => array( 'color', 'background' ),
-		'background-image'                  => array( 'background', 'backgroundImage' ),
+		'background-image'                  => array(
+			array( 'color', 'gradient' ),
```

Comment:
Order matter here, determining the order of values within the property, e.g., 

```css
    background-image: linear-gradient(180deg, rgba(217, 97, 61, 0.69) 0%, rgba(250, 250, 250, 0.24) 100%), url(http://localhost:8888/wp-content/uploads/2024/04/Westwood-Pizza-newtown-1.jpg);
```

The background image doesn't yet officially support opacity but the gradient does, so I think it's fine to keep this order for now.

Let's assume the background image does support opacity for a minute: the order would have to be specified somehow, perhaps by an index or `foreground/background` property in `backgroundImage`?

Thinking further ahead, `background-image` also supports multiple images. The order in which they appear in theme.json or via some sorting UI control should suffice here.